models/network-node.go

```diff
@@ -61,6 +62,8 @@ type AgencyNode struct {
 	RuiRegistration    time.Time             `json:"ruiRegistration" firestore:"ruiRegistration" bigquery:"-"`
 	BigRuiRegistration bigquery.NullDateTime `json:"-" firestore:"-" bigquery:"ruiRegistration"`
 	Skin               *Skin                 `json:"skin,omitempty" firestore:"skin,omitempty" bigquery:"-"`
+	Pec                string                `json:"pec,omitempty" firestore:"pec,omitempty" bigquery:"pec"`
+	Website            string                `json:"website,omitempty" firestore:"website,omitempty" bigquery:"website"`
```

Comment:
do we need to save these fields is a column in bigquery tables?

Reply:
I'm all for saving everything in big query to avoid missing fields for future queries/analysis

Reply:
Deleted the fields from bigquery